diff --git a/node_modules/expo-modules-core/android/ExpoModulesCorePlugin.gradle b/node_modules/expo-modules-core/android/ExpoModulesCorePlugin.gradle
index 4ec4514..292f0f0 100644
--- a/node_modules/expo-modules-core/android/ExpoModulesCorePlugin.gradle
+++ b/node_modules/expo-modules-core/android/ExpoModulesCorePlugin.gradle
@@ -92,7 +92,10 @@ ext.useExpoPublishing = {
     publishing {
       publications {
         release(MavenPublication) {
-          from components.release
+          def releaseComponent = components.findByName("release")
+          if (releaseComponent != null) {
+            from releaseComponent
+          }
         }
       }
       repositories {
